htdocs/main.inc.php

```diff
@@ -133,7 +133,7 @@ function testSqlAndScriptInject($val, $type)
 	$inj += preg_match('/onmouse([a-z]*)\s*=/i', $val);       // onmousexxx can be set on img or any html tag like <img title='...' onmouseover=alert(1)>
 	$inj += preg_match('/ondrag([a-z]*)\s*=/i', $val);        //
 	$inj += preg_match('/ontouch([a-z]*)\s*=/i', $val);        //
-	$inj += preg_match('/on(abort|afterprint[beforeprint|beforeunload|blur|canplay|canplaythrough|change|click|contextmenu|copy|cut)\s*=/i', $val);
+	$inj += preg_match('/on(abort|afterprint[beforeprint|beforeunload|blur|canplay|canplaythrough|change|click|contextmenu|copy|cut])\s*=/i', $val);
```

Comment:
```suggestion
	$inj += preg_match('/on(abort|afterprint|beforeprint|beforeunload|blur|canplay|canplaythrough|change|click|contextmenu|copy|cut)\s*=/i', $val);
```

Maybe I am wrong but shouldn't be like this ?

Reply:
you'r dam right !

PR update